docs/getting-started/cases-req.asciidoc

```diff
@@ -1,11 +1,44 @@
 [[case-permissions]]
 = Cases prerequisites
 
-To view cases, you need the {kib} space `Read` privilege for the `Security` feature. To create cases and add comments, you need the `All` {kib}
-space privilege for the `Security` feature.
+//To view cases, you need the {kib} space `Read` privilege for the `Security` feature. To create cases and add comments, you need the `All` {kib}
+//space privilege for the `Security` feature.
 
-For more information, see
+//For more information, see
+//{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].
+
+You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space and full or partial access can be given. For more information, see
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].
 
 NOTE: To send cases to external systems, you need the
-https://www.elastic.co/subscriptions[appropriate license].
\ No newline at end of file
+https://www.elastic.co/subscriptions[appropriate license].
```

Comment:
Can this provide a bit more info on the type of license, or maybe the specific privilege required (using wording from the table on the target page)? That might save the user a click, or at least help them navigate the large table.

Something like "...you need an [appropriate license](https://www.elastic.co/subscriptions) that allows detection rule external actions" (or whatever the specific priv is from the list).

Reply:
Completely agree with your suggestion, but am going to table this for now since I think this is the start of a larger convo.